### PR TITLE
Make packaging for Flatpak easier

### PR DESCRIPTION
**Pull request by Alexander-Wilms**
This would make the patch at https://github.com/Alexander-Wilms/flathub/tree/new-pr superfluous.

### CODE REVIEW

File: flameshot.pro [hunk 1]
```diff
@@ -198,24 +198,26 @@ RESOURCES += \
...
-    qmfile.path = $${BASEDIR}/usr/share/flameshot/translations/
+    qmfile.path = $${BASEDIR}$${USRPATH}/share/flameshot/translations/
     qmfile.files = $${TRANSLATIONS_FILES}
 
-    dbus.path = $${BASEDIR}/usr/share/dbus-1/interfaces/
+    dbus.path = $${BASEDIR}$${USRPATH}/share/dbus-1/interfaces/
     dbus.files = dbus/org.dharkael.Flameshot.xml
     
-    icon.path = $${BASEDIR}$${USRPATH}/share/icons/
+    icon.path = $${BASEDIR}$${USRPATH}/share/icons/hicolor/scalable/apps
```
Comment (Alexander-Wilms) [inline flameshot.pro]: Not sure if scalable is the correct directory. But when I put the icon in .../hicolor/100x100/apps/, it didn't show up under KDE.

File: flameshot.pro [hunk 2]
```diff
@@ -198,36 +198,38 @@ RESOURCES += \
...
-    qmfile.path = $${BASEDIR}/usr/share/flameshot/translations/
+    qmfile.path = $${BASEDIR}$${PREFIX}/share/flameshot/translations/
     qmfile.files = $${TRANSLATIONS_FILES}
 
-    dbus.path = $${BASEDIR}/usr/share/dbus-1/interfaces/
+    dbus.path = $${BASEDIR}$${PREFIX}/share/dbus-1/interfaces/
     dbus.files = dbus/org.dharkael.Flameshot.xml
     
-    icon.path = $${BASEDIR}$${USRPATH}/share/icons/
+    icon.path = $${BASEDIR}$${PREFIX}/share/icons/hicolor/scalable/apps
```
Comment (Alexander-Wilms) [inline flameshot.pro]: Not sure if scalable is the correct directory. But when I put the icon in .../hicolor/100x100/apps/, it didn't show up under KDE.

Comment (lupoDharkael) [reply flameshot.pro]: could you leave the icon path as it was before?
I'm considering the option of having multiple sized png icons and that will require to revisit where to save them.

Comment (Alexander-Wilms) [reply flameshot.pro]: Yes, I've changed it back

**Pull request merged**